Parse/src/main/java/com/parse/ParseObject.java

```diff
@@ -4237,7 +4246,26 @@ public void writeToParcel(Parcel dest, int flags) {
 
   /* package */ void writeToParcel(Parcel dest, ParseParcelEncoder encoder) {
     synchronized (mutex) {
-      // Write className and id regardless of state.
+      // Developer warnings.
+      ldsEnabledWhenParceling = Parse.isLocalDatastoreEnabled();
+      boolean saving = hasOutstandingOperations();
+      boolean deleting = isDeleting || isDeletingEventually > 0;
+      if (saving) {
+        Log.w(TAG, "About to parcel a ParseObject while a save / saveEventually operation is " +
```

Comment:
I guess depends on developer habits, but generally this should be rare (a configuration change while the object is in the middle of a save).

I agree the message is too long, but I could not make it shorter... Maybe if we merge this and update docs, we can replace with `About to parcel a ParseObject with ongoing tasks. This might be dangerous. Read <link to docs>`?